tools/test-runner-cli.ts

```diff
@@ -35,11 +37,14 @@ const json = stripJsonComments(jsonWithComments, {whitespace: true});
 const whitelist = JSON.parse(json) as Test.WhiteList;
 logger.verbose('TestRunnerCli.Init', `Loading whitelist... DONE`);
 
-const DEFAULT_BACKENDS: ReadonlyArray<TestRunnerCliArgs.Backend> = ['cpu', 'wasm', 'webgl'];
+// The default backends and opset version lists. Those will be used in suite tests.
+const DEFAULT_BACKENDS: ReadonlyArray<TestRunnerCliArgs.Backend> =
+    args.env === 'node' ? ['cpu', 'wasm'] : ['cpu', 'wasm', 'webgl'];
+const DEFAULT_OPSET_VERSIONS: ReadonlyArray<number> = [10, 9, 8, 7];
```

Comment:
Do we need to keep upgrading this after each onnx release too ? If so, I think we better document this...

Reply:
the answer is 'yes' ideally. but considering the fact that node test cases are rarely modified ( usually the change is to add cases to that folder), we can do this as demand.

there is another option: only support the stable ONNX version. This means we should only support v10 after ONNX v1.5 release (which will happen very soon though). thus we only update when a ONNX release happen